### DISCUSSION TITLE
Alerting

### DISCUSSION THREAD

**Discussion by jonwalz** [jonwalz/crypto-data#16] [General]
I keep thinking about how the alerting system is actually going to work. We may need a backend to manage this.

I've read that Pusher is a good tool for managing this: 
https://pusher.com/

We should look up other tools as well.

**Comment by jonfreihofer** [comment]
Yeah, that's why I was researching GraphQL subscriptions before. Something like an event stream/subscription setup, which if I'm not mistaken would have to be defined in some sort of backend code. I'll look into Pusher though

**Comment by jonwalz** [comment]
@jonfreihofer Just found this too https://www.messagebird.com/en/
